### PR TITLE
Update on Auto-escape only the HTML part

### PR DESCRIPTION
**Pull request by tuliolages**
Closes and updates the broken tests from https://github.com/vintasoftware/django-templated-email/pull/120 and also adds a test for the new feature.

### CODE REVIEW

File: tests/backends/test_vanilla_django_backend.py [hunk 1]
```diff
@@ -110,6 +122,16 @@ def test_render_inheritance_email(self):
         self.assertEqual(PLAIN_RESULT, response['plain'])
         self.assertEqual('Another subject for vintasoftware', response['subject'])
 
+    def test_email_text_escaping(self):
+        self.context['username'] = '<p>vintasoftware</p>'
+        self.context['subject'] = 'Subject with tag <p>'
```
Comment (fjsj) [inline tests/backends/test_vanilla_django_backend.py:127]: I think this is not being used/asserted

**Review by fjsj — CHANGES_REQUESTED** [review]
Overall looking good, please check my comment

File: tests/backends/test_vanilla_django_backend.py (hunk 1 above)
Comment (tuliolages) [reply tests/backends/test_vanilla_django_backend.py:127]: It is used as argumento to the method _render_email right below and it changes the response content

**Review by fjsj — APPROVED** [review]
LGTM

File: tests/backends/test_vanilla_django_backend.py (hunk 1 above)
Comment (tuliolages) [reply tests/backends/test_vanilla_django_backend.py]: actually the subject really wasn't being used

**Pull request merged**